scripts/setup-docker-builder

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+set -ex
+
+source ./scripts/version.sh
+
+BUILDER=${PROG}
+
+docker buildx use ${BUILDER}|| \
+    docker buildx create --name=${BUILDER} --platform=linux/amd64,linux/arm64
+
+docker buildx use ${BUILDER}
```

Comment:
What is the reason for setting up this buildx builder vs using the generic setup-buildx action? Is it just to maintain everything inside `/script`?
```
        name: Set up Docker Buildx
        uses: docker/setup-buildx-action@v3
```

Reply:
I think you're right, setup-buildx is a composite step being run already, and it should use a builder automatically.